src/Microsoft.Data.Analysis/Grouping.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Data.Analysis
+{
+    public class Grouping<TKey> : IGrouping<TKey, DataFrameRow>
```

Comment:
Can you add some xml documentation to all new public API?

Reply:
As GroupBy exposes only IGrouping interface and not Grouping<TKey> class, I changed it to be an internal class inside GroupBy<TKey>. I decided it's better, that increase the amount of public entities, if it is not really required

Reply:
In the current form, it doesn't even need to be `internal`. It can just be `private`.

Reply:
Fixed